src/Forms/Styles/SubStationAlphaStyles.cs

```diff
@@ -625,7 +625,7 @@ private void buttonCopy_Click(object sender, EventArgs e)
             {
                 string styleName = listViewStyles.SelectedItems[0].Text;
                 SsaStyle oldStyle = GetSsaStyle(styleName);
-                SsaStyle style = GetSsaStyle(styleName);
```

Comment:
Note: `GetSsaStyle(...)` will call `AdvancedSubStationAlpha.GetSsaStyle(styleName, _header)` which is very expensive